clients/deltalake/src/main/scala/org/projectnessie/deltalake/NessieLogStore.scala

```diff
@@ -364,7 +365,11 @@ class NessieLogStore(sparkConf: SparkConf, hadoopConf: Configuration)
   }
 
   def pathToKey(path: String): ContentKey = {
-    val parts = path.split("/").toList
+    // we can't simply do a path.toString.split("/") as that would result with
+    // "/tmp/a/b/c in a string array where the first element is an empty string,
+    // which isn't supported by ContentKey
+    val p = StringUtils.stripEnd(StringUtils.stripStart(path, "/"), "/")
+    val parts = p.split("/").toList
```

Comment:
do we maybe have a utils class in deltalake module where we could put `buildContentKeyFromFilePath` or the like.
or make this method reusable from the test?

Reply:
we don't have one, but I'll add one in a follow-up with some tests